tilequeue/process.py

```diff
@@ -472,18 +472,16 @@ def format_coord(
 
     formatted_tiles = []
     for cut_coord in cut_coords:
-        cut_scale = _calculate_scale(scale, coord, nominal_zoom)
-
         if cut_coord == coord:
             # no need for cutting if this is the original tile.
             tiles = _format_feature_layers(
                 processed_feature_layers, coord, nominal_zoom, formats,
-                unpadded_bounds, cut_scale, buffer_cfg)
+                unpadded_bounds, scale, buffer_cfg)
```

Comment:
I think it's worth a comment that you're not using cut scale anymore

Reply:
> I think it's worth a comment that you're not using cut scale anymore

added in the next commit